src/main/java/io/cdap/plugin/ariba/source/config/AribaPluginConfig.java

```diff
@@ -63,11 +63,11 @@ public class AribaPluginConfig extends ReferencePluginConfig {
   private final String systemType;
 
   @Macro
-  @Description("Realm name for the customer.")
+  @Description("Target Realm name from which data is to be extracted.")
   private final String realm;
 
   @Macro
-  @Description("Name of the view template you wish to create.")
+  @Description("Name of the View Template from which data to be extracted.")
```

Comment:
add "is". View Template from which data is to be extracted.

Reply:
Added